server/src/main/java/org/elasticsearch/index/get/GetResult.java

```diff
@@ -40,6 +42,9 @@
 
 public class GetResult implements Writeable, Iterable<DocumentField>, ToXContentObject {
 
+    private static final String TYPE_FIELD_NAME = "_type";
+    private static final Text SINGLE_MAPPING_TYPE = new Text(MapperService.SINGLE_MAPPING_NAME);
```

Comment:
Small comment, I think we can use `MapperService.SINGLE_MAPPING_NAME` directly and avoid this extra constant.